components/welcome/index.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { TextSampleIcon } from '@/public/icons';
+import SMClickAnimation from '../click-animation';
+
+const texts = [
+  'Once your PR is merged, the token list will update automatically to include your token and your token will be available on the Base Bridge.',
+  'To complete the migration, submit a pull request to the Superchain token repo list.',
+  'For support please reach out to us via our telegram',
```

Comment:
For support please reach out to us via our telegram channel: https://t.me/+8vDPDkrN_-gwZTA8

Reply:
Done